net/http/authz/store.go

```diff
@@ -75,6 +75,48 @@ func (s *Store) Save(ctx context.Context, g *Grant) sinkdb.Op {
 	)
 }
 
+// SaveAll returns an Op to store all the grants
+// Duplicates are ignored
+// It also sets field CreatedAt to the time the grants
+// are stored, or the time the original grant was stored
+func (s *Store) SaveAll(ctx context.Context, grants []*Grant, policy string) sinkdb.Op {
```

Comment:
Would it be reasonable to extend Save to accept multiple grants?

```go
func (s *Store) Save(ctx context.Context, g ...*Grant) sinkdb.Op
```

I think the implementation would be nearly identical to this SaveAll method (including requiring all provided grants to be in the same policy), but then we wouldn't also need the old Save method in addition. What do you think?